lib/bolt/transport/base.rb

```diff
@@ -162,6 +162,8 @@ def batches(targets)
         targets.map { |target| [target] }
       end
 
+      # Deprecated this no longer verifies the task is from the API
```

Comment:
Do we have another way of determining if a task is from the api? We might want to just add an 'api: true' key to the json data

Reply:
in the future if the task class is the PuppetServer subclass it's from the API.

Reply:
So change it to `task.class == Bolt::Task::Puppetserver` ?

Reply:
it cannot be changed as part of this PR since it's still used to handle older tasks. Overall I don't think it's a good practice to have a helper here we're not using.  Especially in this case where adding logic into the transports that cares whether the task is from the API should be avoided if at all possible.